photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java

```diff
@@ -592,7 +593,10 @@ private void consumeResult(CVPipelineResult result) {
 
     private void consumePipelineResult(CVPipelineResult result) {
         for (var dataConsumer : resultConsumers) {
-            dataConsumer.accept(result);
+            if (!(result instanceof CalibrationPipelineResult)
```

Comment:
This feels way too leaky imo. Does this logic belong in NtDataPublisher instead?

Reply:
I guess it could. Do we want to not update or post everything as blank/no data?